kombu/asynchronous/aws/connection.py

```diff
@@ -170,6 +170,11 @@ def _mexe(self, request, sender=None, callback=None):
 class AsyncAWSQueryConnection(AsyncConnection):
     """Async AWS Query Connection."""
 
+    STATUS_CODE_OK = 200
+    STATUS_CODE_REQUEST_TIMEOUT = 408
+    STATUS_CODE_NETWORK_CONNECT_TIMEOUT_ERROR = 599
+    STATUS_CODES_TIMEOUT = (STATUS_CODE_REQUEST_TIMEOUT, STATUS_CODE_NETWORK_CONNECT_TIMEOUT_ERROR)
```

Comment:
What's the benefit of a tuple instead of a list?

Reply:
Using a tuple results in a smaller memory footprint. They are also faster to allocate on CPython as far as I recall.
Granted, it's only two items but there's no need to use more memory when you can avoid it.